### PR TITLE
Register patient tests

### PR DESCRIPTION
**Pull request by sacuribraimo**
@drryanjames your comments on  #638 about registerPatientPage are addressed here.

### CODE REVIEW

File: test/e2eTests/tests/registerPatient_test.js [hunk 1]
```diff
@@ -22,28 +39,250 @@ Scenario('Validate tab sequence', (I, RegistrationDashboardPage) => {
   I.say(`${LOG_TAG} Click on the New Patient button`);
   registrationPage.clickNewPatienButton();
```
Comment (drryanjames) [inline test/e2eTests/tests/registerPatient_test.js]: This function should return the registerPatientPage. See example [here](https://github.com/esaude/esaude-emr-poc/blob/master/test/e2eTests/pages/loginPage.js#L38)

File: test/e2eTests/tests/registerPatient_test.js [hunk 2]
```diff
@@ -22,28 +39,250 @@ Scenario('Validate tab sequence', (I, RegistrationDashboardPage) => {
...
+  I.see(patient.identifiers[0].identifier3);
+  I.see(patient.person.names[0].givenName);
+  I.see(patient.person.names[0].familyName);
+})
+
+// Clicks the next button without filling required fields
+const validateRequiredFields = (I, RegisterPatientPage, step, numOfVisibleElements) => {
+  const errorMessageElement = '[ng-message="required"]';
+
+  I.say(`${LOG_TAG} Try to skip the ` + step);
```
Comment (drryanjames) [inline test/e2eTests/tests/registerPatient_test.js]: include step in the comment like this: `... ${step}`

**Comment by drryanjames** [comment]
You'll probably want to reset your fork next time so your old commits don't show up. To reset your fork do something like...
```git reset HEAD upstream/master```
...then add your new changes

File: test/e2eTests/tests/registerPatient_test.js (hunk 2 above)
Comment (sacuribraimo) [reply test/e2eTests/tests/registerPatient_test.js]: Missed that one. Thanks

File: test/e2eTests/tests/registerPatient_test.js (hunk 1 above)
Comment (sacuribraimo) [reply test/e2eTests/tests/registerPatient_test.js]: It navigates to that page. So, yes.

**Pull request merged**